README.md

```diff
@@ -481,20 +494,27 @@ go run main.go --config=$config.yaml
 Below is the sample config file(the tags for the images can be edited as required):
 ```bash
 Images:
-  ApiServer: quay.io/opendatahub/ds-pipelines-api-server:main-0e8a011
-  Artifact: quay.io/opendatahub/ds-pipelines-artifact-manager:main-0e8a011
+  ApiServer: quay.io/opendatahub/ds-pipelines-api-server:latest
+  Artifact: quay.io/opendatahub/ds-pipelines-artifact-manager:latest
   OAuthProxy: registry.redhat.io/openshift4/ose-oauth-proxy:v4.12.0
-  PersistentAgent: quay.io/opendatahub/ds-pipelines-persistenceagent:main-0e8a011
-  ScheduledWorkflow: quay.io/opendatahub/ds-pipelines-scheduledworkflow:main-0e8a011
+  PersistentAgent: quay.io/opendatahub/ds-pipelines-persistenceagent:latest
+  ScheduledWorkflow: quay.io/opendatahub/ds-pipelines-scheduledworkflow:latest
   Cache: registry.access.redhat.com/ubi8/ubi-minimal
   MoveResultsImage: registry.access.redhat.com/ubi8/ubi-micro
   MariaDB: registry.redhat.io/rhel8/mariadb-103:1-188
-  MlmdEnvoy: quay.io/opendatahub/ds-pipelines-metadata-envoy:1.7.0
-  MlmdGRPC: quay.io/opendatahub/ds-pipelines-metadata-grpc:1.0.0
-  MlmdWriter: quay.io/opendatahub/ds-pipelines-metadata-writer:1.1.0
+  MlmdEnvoy: quay.io/opendatahub/ds-pipelines-metadata-envoy:latest
+  MlmdGRPC: quay.io/opendatahub/ds-pipelines-metadata-grpc:latest
+  MlmdWriter: quay.io/opendatahub/ds-pipelines-metadata-writer:latest
   CRDViewer: gcr.io/ml-pipeline/viewer-crd-controller:2.0.0-rc.2
```

Comment:
let's remove these 2 images, as I don't believe we'll be using introducing them now (from unrelated spikes/decisions during this sprint)